File: daguilar9.c
// Daira Aguilar
//


#include<stdio.h>
#include<stdlib.h>
#include<time.h>

int main(void)
{
	int guess , i, x= 3, y;

        srand(time(NULL));


	y = 1 +(rand() %20);

	printf("Enter a number between 1 and 20\n");

	while(guess < 1 || guess > 20 )
	{
	//	printf("Enter a number between 1 and 20\n");
		scanf("%d",&guess);
	
	
		if(guess == y) 
		{
		for(i = 0; i < guess; i++)
		{
		printf("You won and guessed correctly!. This is y = %d\n", y);
		}
		}
	       	else if (guess < y)
		{
		printf("You lost and your guess is lower than my number, %d\n", y);
		}
		else if (guess > y)
		{
		printf("You lost and your guess is higher than my number, %d\n", y);

		 if(guess <  1 || guess > 20 )
                {
                        puts("your imput is out of range(1-20) re-enter value: ");
                }
	}
}
	return EXIT_SUCCESS;

}

File: daguilar9.java
// Daira Aguilar
//

import java.util.Scanner;
import java.util.Random;


public class daguilar9
{
	public static void main(String[] args)
	{
		Random rand = new Random();
		int randNumber = rand.nextInt(20) + 1;

		Scanner scanner = new Scanner(System.in);
		int guess = 0;

		System.out.println("Guess a number between 1 and 20: ");

		while (guess < 1 || guess > 20)
		{
			guess = scanner.nextInt();

			if(guess == randNumber)
			{
				for(int i = 0; i < guess; i++)
				{
					System.out.println("You won and guessed the number correctly !");
				}
			} else if ( guess < randNumber)
			{
				System.out.println("You lost and your guess is lower than my number, " + randNumber);
			} else if (guess > randNumber)
			{
				System.out.println("You lost and your guess is higher than my number, " + randNumber);
			}

			if (guess < 1 || guess > 20)
			{
				System.out.println("Your input is a number that is out of range (1-20). Try again : ");
			}
		}
	}
}
